### PR TITLE
Add PhononJs@1.4.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9406

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/quark-dev/Phonon-Framework
 * Official website (optional, not the repository): http://phonon.quarkdev.com/
 * License and its reference: [MIT](https://github.com/quark-dev/Phonon-Framework/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 25
   - Count of stars: 282
   - Count of forks: 71

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

Note that homepage is `http` rather than `https`.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 please help me review this pull request, thank you.

File: ajax/libs/PhononJs/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "PhononJs",
+  "filename": "js/phonon-core.min.js",
+  "version": "1.4.0",
+  "description": "Phonon is a lightweight & scalable Javascript/CSS3/HTML5 mobile framework to develop a single page application based on Phonegap / Apache Cordova.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/quark-dev/Phonon-Framework.git"
+  },
+  "keywords": [
```
Comment (hare1039) [inline ajax/libs/PhononJs/package.json]: Some key word missing from upstream
https://github.com/quark-dev/Phonon-Framework/blob/master/bower.json#L8

Comment (hare1039) [inline ajax/libs/PhononJs/package.json]: Some key word missing from upstream
https://github.com/quark-dev/Phonon-Framework/blob/master/bower.json#L8

**Comment by extend1994** [comment]
@hare1039 I modified the `keywords` but not exactly the same as `bower.json` cause search the lib with the one in `bower.json` can't get the better search results. Please help me review it again, thank you.

**Comment by extend1994** [comment]
ping @cdnjs/intern3

**Comment by extend1994** [comment]
Close accidentally and reopen it.

**Comment by extend1994** [comment]
@hare1039 @sufuf3 I confirmed that we don't need `theme.css`. Except this, we need all file under `dist`. Please help me review it again, thanks.

**Comment by extend1994** [comment]
ping @sufuf3 @hare1039

**Comment by PeterDaveHello** [comment]
@extend1994 you should better give reference about the `theme.css` directly in the comment and rebase the branch first before ping reviewer next time.
@sufuf3 @hare1039 don't forget to ask for rebase please.

**Comment by PeterDaveHello** [comment]
I'll rebase this branch this time to speed up the whole process.

**Pull request merged**